nemo/collections/nlp/models/glue_benchmark/glue_benchmark_model.py

```diff
@@ -98,21 +99,31 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
             cfg.validation_ds.file_name = os.path.join(self.data_dir, cfg.validation_ds.file_name)
         logging.info(f'Using {cfg.validation_ds.file_name} for model evaluation.')
 
-        self.tokenizer = get_tokenizer(
-            tokenizer_name=cfg.language_model.tokenizer,
-            pretrained_model_name=cfg.language_model.pretrained_model_name,
-            vocab_file=cfg.language_model.vocab_file,
-            tokenizer_model=cfg.language_model.tokenizer_model,
-            do_lower_case=cfg.language_model.do_lower_case,
-        )
+        if cfg.language_model.config_file is not None:
+            logging.info(
+                (
+                    f"HuggingFace BERT config file found. "
+                    f"LM will be instantiated from: {cfg.language_model.config_file}"
+                )
+            )
+            self.vocab_size = json.load(open(cfg.language_model.config_file))['vocab_size']
+        elif cfg.language_model.config and cfg.language_model.config.vocab_size is not None:
+            self.vocab_size = cfg.language_model.config.vocab_size
+        else:
+            self.vocab_size = None
+
```

Comment:
why this can't be a part of get_lm_model?

Reply:
will remove entire part since tokenizer vocab_size will be removed